plugins/CoreHome/vue/src/EnrichedHeadline/EnrichedHeadline.vue

```diff
@@ -45,7 +45,7 @@
         :class="{ 'active': showInlineHelp }"
         :title="translate(reportGenerated ? 'General_HelpReport' : 'General_Help')"
       ><span class="icon-info" /></a>
-      <div class="ratingIcons">
+      <div class="ratingIcons" v-show="showRateFeature">
```

Comment:
Is there any particular reason to use `v-show`? Does the UI layout break if the element is removed completely using `v-if`?

```suggestion
      <div class="ratingIcons" v-if="showRateFeature">
```

Reply:
I've no preference here, `v-if` seems to work as just as well as `v-show`  :+1:

Reply:
The difference is that `v-if` removes the markup completely whereas `v-show` just toggles visibility via CSS, so for this issue I think the correct-er way is `v-if`.

Reply:
Yep, always good to avoid extra markup no matter how small :slightly_smiling_face: